install/kubernetes/templates/_helpers.tpl

```diff
@@ -21,6 +24,11 @@ helm.sh/chart: {{ include "tetragon-operator.chart" . }}
 {{ include "tetragon-operator.selectorLabels" . }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
+{{- define "tetragonPod-controller.labels" -}}
+helm.sh/chart: {{ include "tetragonPod-controller.chart" . }}
+{{ include "tetragonPod-controller.selectorLabels" . }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
```

Comment:
```suggestion
```
let's use `tetragon-operator.selectorLabels` instead.

Reply:
Should I remove the `tetragonPod-controller.labels` labels as well ? or just the `tetragonPod-controller.selectroLabels`. Are you suggesting me not create new labels, instead use the pre-existing `tetragon-operator.labels` ?

Reply:
yeah just use the existing `tetragon-operator` helpers and remove all the `tetragonPod-controller` stuff